### ISSUE TITLE
Laser Pointer

### ISSUE THREAD

**Issue by Alalalalaki** [SlidePilot/SlidePilot-macOS#62]
Is there any chance to add a laser pointer like the one in GoodNotes?

**Comment by columbbus** [comment]
This is a great feature. But since this is more like drawing it won't be that easy to implement next to the other currently available pointers. It may be easier to implement it as a new style in the drawing mode, which makes lines that automatically disappear.

![image](https://user-images.githubusercontent.com/11176702/128152218-096289bf-0bee-4876-8fe8-3fa81b688ba6.png)

I imagine you could put a new symbol (possibly a laser pointer) in the drawing toolbar, which implements the feature you described.

Do you think this would be a suitable solution?

**Comment by Alalalalaki** [comment]
@columbbus  The problem with drawing mode is that one need to enter and exit drawing mode for every slide, which is a little cumbersome. Also is it possible to improve the smoothness of the draw and allow to choose the size or type of the draw. To be honest I feel the current drawing mode is not that convenient and useful in practice.

**Comment by columbbus** [comment]
That's true, I didn't thought about that. So it would be better to do it like the other pointers, always available.

Do you maybe want to open another issue tagged _improvement_ with your ideas for improving drawing mode?

**Comment by Alalalalaki** [comment]
@columbbus I opened a new issue about the draw.
